environment/container/dockerengine.go

```diff
@@ -231,14 +233,26 @@ func (e *dockerEngine) CopyDirsIntoImage(image, newImageName string, paths map[s
 	return nil
 }
 
-// CopyDirsIntoContainer copies some directories into a container
-func (e *dockerEngine) CopyDirsIntoContainer(containerID string, paths map[string]string) (err error) {
+// CopyDataIntoContainer copies files or directories into a container
+func (e *dockerEngine) CopyDataIntoContainer(containerID string, paths map[string]string) (err error) {
 	for sp, dp := range paths {
-		err = copyDirToContainer(e.ctx, e.cli, containerID, sp, dp)
+        fi, err := os.Stat(sp)
```

Comment:
Fix indentation. Use `go fmt`.
If you are using a text editor like VSCode, VIM, Sublime, etc. installing the Golang extension automatically runs the go formatter when you save the file.